README.md

```diff
@@ -1,123 +1,90 @@
-feedback
-========
+# Feedback
 
-This app handles user feedback related things.
+This is a public facing Ruby on Rails application that collects feedback from users in multiple ways (which are detailed below). This feedback information is then used to raise tickets in Zendesk.
 
-### Testing
+## Contact Page on GOV.UK
+![Contact Page on GOV.UK](/docs/screenshots/contact_page.png?raw=true "Contact Page on GOV.UK")
 
-To run unit tests, execute the following:
+## Contact Form on GOV.UK
+![Contact Form on GOV.UK](/docs/screenshots/contact_form.png?raw=true "Contact Form on GOV.UK")
 
-    bundle exec rake
+### Technical documentation
 
-### Testing with a mock `signon` strategy
+The app handles user feedback in the following ways:
 
-Launch using `bowl` from the `development` directory:
+1. Renders a contact page on GOV.UK: `https://www.gov.uk/contact/govuk`
+2. Collects feedback from other contact forms. Pages using the `static` app will have a link at the bottom of the page asking: `Is there anything wrong with this page?`. Clicking this link will show you a form. The data from the form will be submitted to the `feedback` app.
+3. Raises tickets based on feedback to Zendesk. The app does this by submitting data to the `support` app.
+3. A certain subset of tickets ('assisted digital') are stored twice: they're sent once to Zendesk and once to a Google spreadsheet via the Google API. More documentation about that here: [docs/assisted_digital_feedback.md](docs/assisted_digital_feedback.md).
```

Comment:
These are not sent to zendesk; they're sent to the google spreadsheet, and then a subset of the data is sent to support-api as a bit of service feedback (which is one of the things support-api does just write to its db).